activities/community-calls/preparing-community-call.md

```diff
@@ -43,6 +45,6 @@ The chair and the notetaker:
 * add "updates since the last community call" to the notes - keep this to three items or fewer (we want the community call to be a dialogue, not a broadcast). If possible, focus on Standard related updates, but other news relevant to the community are also fine. All updates should be linkable, meaning that they should already be published somewhere else
```

Comment:
This section should probably be rephrased now when it is not specifically for the Standard community call anymore.